aea/cli/push_all.py

```diff
@@ -39,7 +39,18 @@
 @registry_flag()
 def push_all(packages_dir: Optional[Path], registry: str) -> None:
     """Push all available packages to a registry."""
+    try:
+        push_all_packages(registry, packages_dir)
+    except ValueError as e:
+        raise click.ClickException(str(e)) from e
```

Comment:
I think it would be better to catch the value error earlier, e.g. wrap lines 72-73? Everything else is handled afaict